CentralComputing/Pod_State.cpp

```diff
@@ -209,13 +213,54 @@ void Pod_State::steady_launch_ready(std::shared_ptr<NetworkManager::Network_Comm
 }
 
 void Pod_State::steady_flight_accelerate(std::shared_ptr<NetworkManager::Network_Command> command) {
+	// Access Pos, Vel, and Accel from Motion Model
+	double MAX_DECCEL = 19.6;
+	double IDEAL_DECCEL = 9.8;
+	double LENGTH_OF_TRACK = 1000;
+	double BUFFER_LENGTH = 20;
+	double MAX_VELOCITY = 550;
```

Comment:
Let's move these constants out of the function, put them at the top of this file for now